src/assets/Signup.css

```diff
@@ -0,0 +1,16 @@
+@media all and (min-width: 480px) {
+  .Signup {
```

Comment:
you should use lowercase kebab style(hyphenated) for css selectors. This is a general convention